src/js/iframe/iframe.js

```diff
@@ -64,7 +65,7 @@ const handleMessage = (event: PostMessageEvent): void => {
                 fail('POPUP.HANDSHAKE: popupMessagePort not found');
                 return;
             }
-            _popupMessagePort = event.ports[0];
+            [_popupMessagePort] = event.ports;
```

Comment:
doublecheck

Reply:
doublecheck of what? this is only destructuring, transpiled by babel to:
`
var _event$ports = event.ports;

_popupMessagePort = _event$ports[0];
`